spinalcordtoolbox/deepseg_sc/cnn_models_3d.py

```diff
@@ -13,9 +13,6 @@
 from keras.layers import Conv3D, MaxPooling3D, UpSampling3D, Activation, BatchNormalization
 from keras.optimizers import Adam
 from keras.models import load_model
-
-K.set_image_data_format("channels_first")
-
```

Comment:
Yes good idea.

Reply:
i would be super careful with that! i remember about 1y ago (or so...) I also removed that line because I thought it was useless, and ended up struggling for several days because i've introduced a bug. Indeed it is needed somewhere else, as pointed out in https://github.com/neuropoly/spinalcordtoolbox/issues/2954#issuecomment-710479070. @charleygros is definitely the right person to talk to. I've slacked him, he will help us with this.